job-devrel.rst

```diff
@@ -0,0 +1,50 @@
+:orphan:
+
+Job Posting: Developer relations with documentation skills
+==========================================================
+
+.. |role| replace:: developer relations
+
+.. include:: /hiring/intro.rst
+
+About the job
+-------------
+
+This job will be our first |role| hire.
+You will be responsible for developer marketing and improving our documentation.
+
+We have recently gotten a :doc:`grant to support scientific software</czi-grant-announcement>` in 2021.
+Given this grant, the job will have two major parts:
+
+* For 2021, around half your time will be spent working on work related to our grant.
+  This focuses on improving our guide and tutorials for Read the Docs and supporting software in the ecosystem (Sphinx, docutils)
+* The other half of your time will be spent working on Read the Docs itself, outside of the grant funding.
+  This includes managing a number of improvements to our documentation structure, and improving our `user guides <https://docs.readthedocs.io/en/latest/guides/>`_.
+
+Historically we have had a very developer-focused audience,
+but we are trying to make our documentation more appropriate for less technical users.
+One of our first priorities will be creating more beginner and tutorial content for Read the Docs.
+This could perhaps include some video or audio content,
+along with written docs.
+
+The other major part of this job will be
+
+
+Technical Details
+-----------------
+
+Read the Docs is a large Python & Django web application.
+The current technologies that we're using are:
+
+* Sphinx/reStructuredText for documentation
+* Read the Docs for documentation building & hosting :)
+* Our blog is somewhat custom, built on Sphinx as well
+* Twitter for most of our marketing and developer relations
+
+We historically have chosen overly technical tools, but we are hoping to make things more standardized over time.
+We are planning to move our marketing pages away from our application and into a static site generator,
+and doing similar things with our blog.
+This should make it easier to write and deploy new marketing pages,
+as well as make it easier to onboard people with less technical backgrounds.
+
```

Comment:
We haven't discussed the developer relations aspect of this position much at all in this description. The questions I'd have here, as a reader, are:

* What are our goals for the developer relations work?
* What are the responsibilities of this role?
* What work are we doing here now/where are we in this process/etc?